src/Internal/Wallet/Cip30.purs

```diff
@@ -79,13 +84,15 @@ type Cip30Wallet =
   , getRewardAddresses :: Aff (Array Address)
   , signTx :: Transaction -> Aff Transaction
   , signData :: Address -> RawBytes -> Aff DataSignature
+  , getPubDrepKey :: Aff PublicKey
```

Comment:
Is it always available? Shouldn't it be `Maybe PublicKey`?

Reply:
This key is derived from the account, so I'd assume it should be always available

Reply:
At least I don't see any notes in the CIP-95 about it being optional, let's revisit this concern later

Reply:
OK let's consider it always available